packages/suite/.stylelintignore

```diff
@@ -0,0 +1,6 @@
+# stylelint does not accept re-exporting types (export type {} from ...). throws Unexpected token
```

Comment:
controversial, 
`stylelint` doesnt parse `export type { A } from 'b'` correctly and throws `Unexpected token` error. i didn't find any solution and i don't want to waste my time with it.